src/components/policy/src/policy/policy_table/table_struct/types.cc

```diff
@@ -489,6 +489,24 @@ ModuleConfig::ModuleConfig(const Json::Value* value__)
     vehicle_model(impl::ValueMember(value__, "vehicle_model")),
     vehicle_year(impl::ValueMember(value__, "vehicle_year")) {
 }
+
+void ModuleConfig::SafeCopyFrom(const ModuleConfig& from) {
```

Comment:
Could we use an ``opearator=`` witn additional ``FiledVerification`` method call?